locales/it/index.yml

```diff
@@ -678,6 +678,8 @@ messages:
     senderInfo:  Mittente sconosciuto
     serviceInfo: Info sul servizio mancanti
     departmentInfo: Info sul dipartimento mancanti
+  noConnectionTitle: Offline! Non puoi ricevere messaggi
+  noConnectionDescription: Connettiti per visualizzare i messaggi
```

Comment:
```suggestion
```
I would remove this text in place of a button "riprova" where the user can try to reload the messages

Reply:
> I would remove this text in place of a button "riprova" where the user can try to reload the messages

@Undermaken There is active pull-down to try to re-download the data, it seems superfluous to add the button too. What do you think about it?

Reply:
Yes, we could keep the same behavior of services where to refresh the data the user has to pull down the list